babble/src/main/java/io/mosaicnetworks/babble/servicediscovery/webrtc/WebRTCService.java

```diff
@@ -39,296 +38,163 @@
 
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.net.UnknownHostException;
-import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 
-import io.mosaicnetworks.babble.configure.OnNetworkInitialised;
-import io.mosaicnetworks.babble.discovery.Peer;
 import io.mosaicnetworks.babble.discovery.PeersProvider;
-import io.mosaicnetworks.babble.node.BabbleService;
 import io.mosaicnetworks.babble.node.ConfigManager;
 import io.mosaicnetworks.babble.service.ServiceAdvertiser;
+import io.mosaicnetworks.babble.servicediscovery.ResolvedGroup;
+import io.mosaicnetworks.babble.servicediscovery.ResolvedService;
 import io.mosaicnetworks.babble.servicediscovery.ServiceDiscoveryListener;
 
-// import android.net.wifi.WifiManager;
-
 /**
  * Service class to handle WebRTC
  */
 public class WebRTCService implements ServiceAdvertiser {
 
```

Comment:
Still mourning the BabbleConstants.java approach here. 

public static final constants in the library here are not good for reuse.

Reply:
We can revisit the BabbleConstants approach for UI components. However for non UI components such as this, I think public variables or setter methods is the norm?

Reply:
I know BabbleConstants is a potential future enhancement. The immediate point is that we need to lose the final modifier and add setter methods to allow other WebRTC servers to be used.